src/app/file_manager/file_manager.hpp

```diff
@@ -0,0 +1,28 @@
+#include <filesystem>
+#include <fstream>
+#include <string>
+
+class FileManager
+{
+  public:
+    enum class FileMode
+    {
+        Input,
+        OutputAppend,
+        OutputTruncate
```

Comment:
explicitly assign values

Reply:
done

Reply:
It's a minor detail, but to me more appealing names might be something like: Read, WriteAppend, WriteTruncate or just Read, Append, Truncate